### ISSUE TITLE
Move to published `@voxpelli/semver-set`

### ISSUE THREAD

**Issue by voxpelli** [anvaka/npmrank#27]
I noticed that this project is depending on a git commit of my module (added in #10) rather the now published version of it.

Also: Its now no longer a fork but a rewritten standalone project that preserves compatibility: https://www.npmjs.com/package/@voxpelli/semver-set

And since I use it myself in [installed-check-core](https://github.com/voxpelli/node-installed-check-core) I'm maintaining it and ensuring it keeps working.

**Comment by anvaka** [comment]
Thank you for the heads up!

I tried to quickly switch to your version, but running into compatibility issues:

```
/Users/anvaka/projects/ngraph/experiments/npmrank/countVersions.js:2
var intersect = require('@voxpelli/semver-set').intersect;
                ^

Error [ERR_REQUIRE_ESM]: require() of ES Module 
```

**Comment by voxpelli** [comment]
Right, yeah, you would have to do `var { intersect } = await import('@voxpelli/semver-set');`, but that would be hard as its currently used in a non-async context :/

Sorry for making it an ESM module
